deployments/values/supported-matrix.yaml

```diff
@@ -110,32 +104,70 @@ matrix:
     tengine-pdf: *tengines_ga_version
     tengine-tika: *tengines_ga_version
     activiti:
-      version: &activiti_ga_version "24"
+      version: &activiti_ga_version "25"
       pattern: *ga_pattern
     activiti-admin:
       version: *activiti_ga_version
       pattern: *ga_pattern
-    audit-storage:
-      version: "~1.0.0"
+    audit-storage: &audit_storage_ga_version_spec
+      version: "~1.0"
+      versionFilterKind: semver
+
+  23.N:
+    id: 23n
+    acs: &acs_23_version_spec
+      version: "~23"
       versionFilterKind: semver
+    activemq: *activemq_ga_version_spec
+    share: *acs_23_version_spec
+    search: *search_ga_version_spec
+    insight-zeppelin: *search_ga_version_spec
+    search-enterprise:
+      version: "4.2"
+      pattern: *ga_hotfixes_pattern
+    sync:
+      version: "5.1"
+      pattern: *ga_hotfixes_pattern
+    adw:
+      version: "5.3"
+      pattern: *ga_pattern
+    adminApp:
+      version: "9.3"
+      pattern: *ga_pattern
+    onedrive:
+      version: "2"
+      pattern: *ga_hotfixes_pattern
+    msteams:
+      version: "2"
+      pattern: *ga_hotfixes_pattern
+    intelligence: *ais_ga_version
+    trouter: *ats_ga_version
+    sfs: *ats_ga_version
+    tengine-aio: *tengines_ga_version
+    tengine-misc: *tengines_ga_version
+    tengine-im: *tengines_ga_version
+    tengine-lo: *tengines_ga_version
+    tengine-pdf: *tengines_ga_version
+    tengine-tika: *tengines_ga_version
+    activiti:
+      version: &activiti_24_version "24"
+      pattern: *ga_pattern
+    activiti-admin:
+      version: *activiti_24_version
+      pattern: *ga_pattern
+    audit-storage: *audit_storage_ga_version_spec
```

Comment:
23.4. only supports 1.0.0 and 25.1 1.1.0

Reply:
there should be only one supported release and will be the latest available